.gitpod.yml

```diff
@@ -9,12 +9,14 @@ tasks:
     command: |
       export WASP_TELEMETRY_USER_ID=$GITPOD_WORKSPACE_ID
       cd MyNewApp
+      export REACT_APP_API_URL="https://3001-$GITPOD_WORKSPACE_ID.$GITPOD_WORKSPACE_CLUSTER_HOST"
```

Comment:
This is neat! However we also have a command to get the same thing: `$(gp url 3001)`

Reply:
Also, I would recommend to these variables globally. Right now these env vars are scoped within the task terminal.

You could do this from a [custom dockerfile](https://www.gitpod.io/docs/config-docker#configure-a-custom-dockerfile):

I will push a commit.